internal/fields/validate.go

```diff
@@ -1256,6 +1256,25 @@ func (v *Validator) parseSingleElementValue(key string, definition FieldDefiniti
 	return nil
 }
 
+// IsDocumentation reports whether ip is a reserved address for documentation,
+// according to RFC 5737 (IPv4 Address Blocks Reserved for Documentation) and
+// RFC 3849 (IPv6 Address Prefix Reserved for Documentation).
+func IsDocumentation(ip net.IP) bool {
```

Comment:
Nit. Consider adding some test case in `validate_test.go` and/or in some test package under `test/packages`.